### ISSUE TITLE
Negative filters for categories

### ISSUE THREAD

**Issue by Lulushu12** [ModOrganizer2/modorganizer#553]
Hi guys, just ran into an issue today and I managed to find a workaround, but I realized it would have been far easier to handle if MO just had a "negative filter" feature. What I mean by that is that it would be nice to have a feature that in addition to "and" and "or" it would also say "not". So by clicking that and, say, "contains plugins", it would show all the mods that don't have any plugins. I don't know if this was proposed before, so I'm sorry if it was. Thanks for the awesome tool btw

tl;dr: giff negative filters pls

**Comment by Al12rs** [comment]
We already have plans for this feature and is on the road map 😉  (still depends if we actually have free time to code it).
